### ISSUE TITLE
Modify bash quickstart scripts to automatically start Config, Auth and Dict services.

### ISSUE THREAD

**Issue by jzgithub1** [NationalSecurityAgency/datawave#495]
Take the commands for starting the configuration,  authorization and dictionary service from the new Services/README.md file and put them where they belong in the quickstart bash scripts so they start up when we perform 'allStart' command.

**Comment by keith-ratcliffe** [comment]
I hadn't really thought through the best way to implement this yet.

To keep things simple, until we reach the point where query and/or ingest *requires* microservices to be active, I'm thinking it's probably best to leave the default behavior of the quickstart as-is, and provide setup for microservices as optional bootstraps that can be easily toggled, thru whatever mechanism at setup time

FWIW, the Wildfly configuration that'll be required for this can already be toggled via the [variable defined here](https://github.com/NationalSecurityAgency/datawave/blob/master/contrib/datawave-quickstart/bin/services/datawave/bootstrap-user.sh#L9)

**Comment by jzgithub1** [comment]
I agree.  I looked at the scripts and was not seeing this as a trivial task.

**Comment by keith-ratcliffe** [comment]
On one hand, it'd be nice to be able to use docker-compose as the method of bootstrapping all the microservices. The only real downside is that it would force the user to have the additional docker 
dependencies installed/configured, as opposed to just the JDK. Might be a worthwhile trade-off though, I'm on the fence

**Comment by jzgithub1** [comment]
That's a great idea.  Everyone should have Docker.  Its good for them.  I wrote a small bash script and put it in my services directory for the meantime for my private use.  I won't make a pull request with it.
